### PR TITLE
style: redesign team page

### PR DESCRIPTION
**Pull request by qier222**
### Changes:
style: redesign team page

### Screenshots of the change:

CleanShot 2021-02-19 at 17 54 24@2x
CleanShot 2021-02-19 at 17 54 37@2x

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
@juzhiyuan Do we need to sort the list alphabetically? It seems at first the PMC list is sort in that way but later it goes chronologically 🤣

File: website/src/assets/icons/code.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 20l4-16m4 4l4 4-4 4M6 16l-4-4 4-4" />
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/code.svg]: image

Please add a new line here

File: website/src/assets/icons/database.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 7v10c0 2.21 3.582 4 8 4s8-1.79 8-4V7M4 7c0 2.21 3.582 4 8 4s8-1.79 8-4M4 7c0-2.21 3.582-4 8-4s8 1.79 8 4m0 5c0 2.21-3.582 4-8 4s-8-1.79-8-4" />
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/database.svg]: ditto

File: website/src/assets/icons/eye.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M15 12a3 3 0 11-6 0 3 3 0 016 0z" />
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M2.458 12C3.732 7.943 7.523 5 12 5c4.478 0 8.268 2.943 9.542 7-1.274 4.057-5.064 7-9.542 7-4.477 0-8.268-2.943-9.542-7z" />
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/eye.svg]: ditto

File: website/src/assets/icons/puzzle.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M11 4a2 2 0 114 0v1a1 1 0 001 1h3a1 1 0 011 1v3a1 1 0 01-1 1h-1a2 2 0 100 4h1a1 1 0 011 1v3a1 1 0 01-1 1h-3a1 1 0 01-1-1v-1a2 2 0 10-4 0v1a1 1 0 01-1 1H7a1 1 0 01-1-1v-3a1 1 0 00-1-1H4a2 2 0 110-4h1a1 1 0 001-1V7a1 1 0 011-1h3a1 1 0 001-1V4z" />
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/puzzle.svg]: ditto

File: website/src/assets/icons/terminal.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 9l3 3-3 3m5 0h3M5 20h14a2 2 0 002-2V6a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z" />
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/terminal.svg]: ditto

File: website/src/pages/help.js [hunk 6]
```diff
@@ -1,6 +1,24 @@
 const React = require("react");
 import Layout from "@theme/Layout";
 import ChevronRight from "../assets/icons/chevron-right.svg";
+import styled from "styled-components";
```
Comment (juzhiyuan) [inline website/src/pages/help.js]: would better put this line before L4

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
image

It seems broken?

**Comment by juzhiyuan** [comment]
> @juzhiyuan Do we need to sort the list alphabetically? It seems at first the PMC list is sort in that way but later it goes chronologically 🤣

Ya, would better do that. We could do it in another PR if necessary.

**Comment by qier222** [comment]
> image
> 
> It seems broken?

The team.md file doesn't have his GitHub username on it, I can't find his avatar.

File: website/src/pages/help.js (hunk 6 above)
Comment (qier222) [reply website/src/pages/help.js]: all fixed

**Comment by juzhiyuan** [comment]
> > image
> > It seems broken?
> 
> The team.md file doesn't have his GitHub username on it, I can't find his avatar.

OK, ignore this one currently.

File: website/src/pages/team.js [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+`;
+const MemberName = styled.div`
+  font-size: 1rem;
+  font-weight: 600;
+  margin-top: 12px;
+  margin-bottom: -4px;
+  line-height: 1rem;
+`;
+const Username = styled.div`
+  font-size: 12px;
```
Comment (liuxiran) [inline website/src/pages/team.js]: May I ask why not use `rem` here? `rem` would be more friendly to fit different display devices

Comment (qier222) [reply website/src/pages/team.js]: Use `rem` is indeed a better option, I will fix it.

**Review by juzhiyuan — APPROVED** [review]
Glad to see this big changes!

**Review by liuxiran — APPROVED** [review]

**Pull request merged**